util/stmtsummary/evicted.go

```diff
@@ -63,6 +83,9 @@ func (ssbde *stmtSummaryByDigestEvicted) AddEvicted(evictedKey *stmtSummaryByDig
 	evictedValue.Lock()
 	defer evictedValue.Unlock()
 
+	ssbde.Lock()
+	defer ssbde.Unlock()
```

Comment:
Why add this lock?

Reply:
to fix data racing between toHistoryDatum() and addEvict()